sections/shared/Layout/AppLayout/Header/MobileUserMenu/MobileMenuModal.tsx

```diff
@@ -11,20 +11,21 @@ import FullScreenModal from 'components/FullScreenModal';
 import Logo from 'sections/shared/Layout/Logo';
 import Links from 'sections/dashboard/Links';
 
-import { menuLinksState } from '../states';
+import { homepageMenuLinksState, menuLinksState } from '../states';
 
 import MobileSubMenu from './MobileSubMenu';
 import { MenuButton, SUB_MENUS } from './common';
 import MobileMenuArrow from 'assets/svg/app/mobile-menu-arrow.svg';
 
 type MobileMenuModalProps = {
+	homepage?: boolean;
 	onDismiss(): void;
 };
 
-export const MobileMenuModal: FC<MobileMenuModalProps> = ({ onDismiss }) => {
+export const MobileMenuModal: FC<MobileMenuModalProps> = ({ homepage, onDismiss }) => {
 	const { t } = useTranslation();
 	const { asPath } = useRouter();
-	const menuLinks = useRecoilValue(menuLinksState);
+	const menuLinks = useRecoilValue(homepage ? homepageMenuLinksState : menuLinksState);
```

Comment:
Not an issue (especially since it didn't start in this PR), but was wondering if there was a specific reason we store links in state, if they're constant.

Reply:
I don't think so. Let's remove it.